### DISCUSSION TITLE
New label like "Needs PR"?

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#2549] [Repository Maintenance] [answered]
In https://github.com/streetcomplete/StreetComplete/issues/2540, @westnordost said,

> The help wanted label is more for things where the app author needs help and/or contribution for this is highly appreciated. Not so much for things that won't be done by me because I consider it of such a low priority. Basically everything here in the issue tracker marked as enhancement is "you are welcome to implement this".

I don't think this is clear for issues that are closed, and it is quite common for you to close issues with something like, "I think this is not worth the work to implement, but I would accept a PR". Sometimes these are marked as wontfix, but not always, and not all issues marked wontfix are this way (some are "…will not accept a PR"). So I think a different label would be appropriate.

I don't think `PR Welcome` would be quite right, since it implies that PRs are *not* welcome on other issues. What about `Needs PR`?

Without either leaving these open or adding a label, I think it is unlikely that anyone will find them.

**Comment by westnordost** [comment]
or "low priority" label

**Reply by smichel17** [reply]
Another option I've used is "Someday/maybe"

**Comment by riQQ** [comment]
There should definitely be a good description for the label (labels have a name and a description field in GitHub). And a link from the `Contributing.md`.

**Comment by westnordost** [comment]
"Blue sky"?

On 4 February 2021 04:56:12 CET, smichel17 <notifications@github.com> wrote:
>Another option I've used is "Someday/maybe"
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/streetcomplete/StreetComplete/discussions/2549#discussioncomment-336901

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by riQQ** [comment]
"Needs contribution"?

**Comment by peternewman** [comment]
> Without either leaving these open or adding a label, I think it is unlikely that anyone will find them.

The projects view could also be used to show them:
https://github.com/streetcomplete/StreetComplete/projects

Or milestones:
https://github.com/streetcomplete/StreetComplete/milestones/

> Without either leaving these open or adding a label, I think it is unlikely that anyone will find them.

Obviously it's up to @westnordost . But logistically it's going to be easier within GitHub if things which haven't been implemented and are deemed acceptable are left open, it's quite odd to tell people to look for things that have been closed to implement! But something like milestones might be the easiest way to hide those sort of things during general usage so the issue tracker isn't flooded with potential features, by only viewing issues with no milestone.

**Comment by smichel17** [comment, marked as answer]
I went with `PR accepted` for now.

edit: …and I immediately don't like it. Still thinking :/

edit2: Updated to `needs PR`. It was my first instinct here (although actually I still like `PR welcome` despite what I said in the post), so I'll stick with it.

**Reply by matkoniecz** [reply]
See https://github.com/streetcomplete/StreetComplete/issues?q=is%3Aopen+is%3Aissue+label%3A%22needs+PR%22 for currently labelled issues (though PR are also welcome for other issues, though in case where someone is assigned or someone is actively working it is better to ask first)

**Reply by peternewman** [reply]
Except as per my comment I believe you actually need to look at https://github.com/streetcomplete/StreetComplete/issues?q=is%3Aissue+label%3A%22needs+PR%22 as I think some of the closed labelled ones haven't been implemented either.

**Reply by matkoniecz** [reply]
Then I removed "needs PR" from some implemented already.
